### PR TITLE
Add access restriction message for screen readers.

### LINKED ISSUE #668: Location-restricted message isn't perceivable by screen reader
The "restricted" message doesn't get read by the screenreader when focus is on the video. 

Add an id to the message div, and add aria-labelledby="id" to the video element.

`<div class="sul-embed-media-access-restricted" id="message1">`

`<video id="sul-embed-media-1" aria-labelledby="message1"...`

### PR DESCRIPTION
**Pull request by tingulfsen**
Closes #668

### CODE REVIEW

**Comment by jvine** [comment]
If you use the same ID for every message in an object that has more than one restricted item, the screen reader may grab the first message with that ID in the DOM, regardless of which one is actually displayed on the page. ChromeVox does this. I think each pair (item + corresponding message) needs a unique ID. There might also be something you could do with `aria-hidden` but I haven't been able to make it work.

File: lib/embed/media_tag.rb [hunk 1]
```diff
@@ -127,7 +128,7 @@ def access_restricted_overlay(stanford_only, location_restricted)
       <<-HTML.strip_heredoc
         #{location_only_overlay(stanford_only, location_restricted)}
         <div class='sul-embed-media-access-restricted-container' data-access-restricted-message>
-          <div class='sul-embed-media-access-restricted'>
+          <div class='sul-embed-media-access-restricted' id='access-restricted-message-div'>
```
Comment (jkeck) [inline lib/embed/media_tag.rb]: You should be able to use `file_index` here to (and above in the `aria-labelledby` attribute) to give a unique id to these elements.

**Comment by jkeck** [comment]
@tingulfsen a branch that you have rebased this on top of (I think) makes the video/audio element `display: none` by default.  If you have capybara assertions that deal w/ the raw HTML (and does not trigger the entire JS stack like a full integration test) then you need to add `visible: false` to the `expect`.

**Comment by ndushay** [comment]
LGTM.  👍

**Comment by jkeck** [comment]
LGTM.  Should we squash any of these commits before 🚢ing?

**Pull request merged**